wordpress-mobile/WordPress-iOS-Shared
# Text: Removes problematic markup from Gutenberg galleries.

In this PR I'm adding some logic to parse away the unordered list markup that surrounds images in Gutenberg authored galleries.  This should remove a list item bullet from appearing next to Gutenberg gallery images in the iOS reader.

To test:
Run the unit test and confirm it passes. 
For the adventurous you might point the iOS app at the commit and rebulid pods. You could also clean your build directory, apply the changes in `RichContentFormatter.swift` to the pod source in the WPiOS project, and then build and run. 

View any post with a Gutenberg gallery. @rachelmcr links to one in https://github.com/wordpress-mobile/WordPress-iOS/issues/10206#issuecomment-436964727 and confirm the bullets do not show and images are correctly sized.

Example:
![simulator screen shot - iphone 8 - 2018-11-13 at 17 33 05](https://user-images.githubusercontent.com/1435271/48450429-b6d92080-e76b-11e8-8bac-d03dd493095e.png)

@etoledom could I trouble you with this one too?
